README.md

```diff
@@ -34,6 +34,8 @@ Install [rnpm](https://github.com/rnpm/rnpm) and run `rnpm link react-native-vid
 
 Or if you have trouble using [rnpm](https://github.com/rnpm/rnpm), make the following additions to the given files manually:
 
+**see below for expansion file usage**
```

Comment:
Please remove this, it's not really needed. If we did this for every point in the realm, we would have notes pointing everywhere